docs/src/main/sphinx/connector/bigquery.md

```diff
@@ -121,10 +158,12 @@ a few caveats:
   - Description
   - Default
 * - `bigquery.project-id`
-  - The Google Cloud Project ID where the data reside.
-  - Taken from the service account
+  - The Google Cloud Project ID where the data reside, 
```

Comment:
maybe better to says something like

The project ID of the Google Cloud account used to store the data, see also